packages/shader-shaderlab/src/shaders/shadingPBR/BTDF.glsl

```diff
@@ -0,0 +1,53 @@
+#ifndef BTDF_INCLUDED
+#define BTDF_INCLUDED
+
+#include "Refraction.glsl"
+
+#ifdef MATERIAL_ENABLE_SS_REFRACTION 
+    sampler2D camera_OpaqueTexture;
+    vec3 evaluateRefraction(SurfaceData surfaceData, BRDFData brdfData, vec3 specularColor) {
+        RefractionModelResult ray;
+        #if defined(REFRACTION_SPHERE) 
+            RefractionModelSphere(-surfaceData.viewDir, surfaceData.position, surfaceData.normal, surfaceData.IOR, surfaceData.thickness, ray);
+        #elif defined(REFRACTION_PLANE)
+            RefractionModelBox(-surfaceData.viewDir, surfaceData.position, surfaceData.normal, surfaceData.IOR, surfaceData.thickness, ray);
+        #elif defined(REFRACTION_THIN)
+            RefractionModelBox(-surfaceData.viewDir, surfaceData.position, surfaceData.normal, surfaceData.IOR, 0.005, ray);
+        #endif
+        //TODO: support cubemap refraction.
+        vec3 refractedRayExit = ray.position;
+
+        // We calculate the screen space position of the refracted point
+        vec4 samplingPositionNDC = camera_ProjMat * camera_ViewMat * vec4( refractedRayExit, 1.0 );
+        vec2 refractionCoords = (samplingPositionNDC.xy / samplingPositionNDC.w) * 0.5 + 0.5;
+
+        // Absorption coefficient from Disney: http://blog.selfshadow.com/publications/s2015-shading-course/burley/s2015_pbs_disney_bsdf_notes.pdf
+       #ifdef MATERIAL_HAS_ABSORPTION
+            vec3 absorptionCoefficient = -log(clamp(surfaceData.attenuationColor, 1e-5f, 1.0f)) / max(1e-5f, surfaceData.attenuationDistance);
```

Comment:
calculate `absorptionCoefficient` in `getSurfaceData` and delete `attenuationColor`、`attenuationDistance` in `SurfaceData`